packages/replayer/src/replayer.ts

```diff
@@ -293,7 +293,11 @@ export const replayEvents: ReplayEventsFn = async (options) => {
       }
     }
   } catch (error) {
-    logger.error("Error thrown during replay", error);
```

Comment:
Maybe still worth logging the error, particularly if it's not a MeticulousTimeoutError? I know it goes to the timeline, but if debugging an issue you may not think to look at the timeline in addition to the terminal logs.

Reply:
Oops, removal of this wasn't intended. reverted in #223